### PR TITLE
Enable DefaultRouter in DEBUG mode and refactor serializers for reference robots/objects

### LINKED ISSUE #52: Fix Scene Serialization when Saving
# Fix Scene Serialization when Saving

### 🐛 Bug Summary
Scene data is not being properly serialized when saving changes in SceneStore, causing data loss or corruption during the save process.

### 🔄 Steps to Reproduce
1. Create Empty Scene ( Backend should be running)
2. Add Robots/objects to scene
3. Click the save button
4. Observe error in console/toast notification

### 🔧 Expected Behavior
- Scene changes should be properly serialized and saved in backend
- No console errors during save process
- User should receive confirmation of successful save

### 📚 Logs and Screenshots
Image

### PR DESCRIPTION
**Pull request by karimsiala**
### 💫 Describe your changes
1. URL Routing with DefaultRouter in DEBUG Mode:

 - DefaultRouter Benefits: Automatically generates URL patterns for viewsets and provides a browsable API root for easier development.
 For Example: 
 http://localhost:8000/library/ref-robots/
image

2. Serializer Refactor:

 - Updated serializers to use PrimaryKeyRelatedField for handling object and robot references, fixing the problem with to save objects/robots in the scene manager

### 📋 Issue ticket number and link

Fixes openroboticmetaverse/mvp-webapp#52

### CODE REVIEW

**Comment by YannickP99** [comment]
Could you explain again please why you switched from Simple to DefaultRouter Because my screen looks the same as the screenshot with the current implementation

File: orom_backend/scene_manager/serializers.py
Comment (YannickP99) [inline orom_backend/scene_manager/serializers.py:1]: As far as i understand and could test it, you want only the id of the reference object/robot right? If that is the case you can just delete the line of code (compare to the Scene-Serializer)
Also why do you not need this information?

**Comment by YannickP99** [comment]
Hey I just understood what problem you faced: that you can't just add a object by its id, but that you would have to add its whole information. Maybe we can adapt the code, so that for POST-requests the api only needs the ID and on GET-requests we also get the information of the reference-object. That should not be too hard, we can easily overwrite the GET and POST methods of the view

**Comment by karimsiala** [comment]
@YannickP99 I removed the lines as we discussed, and I don't think the default router being enabled in debug mode would have much an impact for now. 
If everything is okay we can merge this I suppose

**Review by YannickP99 — APPROVED** [review]

**Pull request merged**